test/unit/resources/elasticsearch_test.rb

```diff
@@ -32,7 +32,7 @@
         _(node_roles).must_include 'master'
       end
     end
-    resource_with_url = load_resource('elasticsearch','http://localhost:9200/_nodes/')
+    resource_with_url = load_resource('elasticsearch','http://localhost:9200/_nodes/ -u es_admin:password')
```

Comment:
This is not what I meant... that's not part of the URL, that's now a command flag you pass to curl, but in the future, this resource may use a different method of fetching an HTTP endpoint.

It's possible to have elasticsearch clusters on different hostnames and different URIs. For example:

`http://mydatacenter.mycompany.biz/myelasticsearch`

... in which case, your resource would fetch from `http://mydatacenter.mycompany.biz/myelasticsearch/_nodes`